lib/Settings.js

```diff
@@ -38,20 +39,31 @@ Settings.loadRepositoriesSettings = function (configPath) {
     }
     repositoryNames = Object.keys(configJson);
     if (repositoryNames.length === 0) {
-        return Promise.reject('`repositories` be non-empty');
+        return Promise.reject('`repositories` must be non-empty');
     }
     for (var i = 0; i < repositoryNames.length; i++) {
-        name = repositoryNames[i];
-        if (!defined(configJson[name].gitHubToken)) {
-            return Promise.reject('repository ' + name + ' must have a `gitHubToken`');
+        full_name = repositoryNames[i];
+        if (!/\//.test(full_name)) {
+            return Promise.reject('repository ' + full_name + ' must be in the form {user}/{repository}');
         }
+        // Modify repository names to be accessed by their short names
+        short_name = full_name.split('/')[1];
+        var tmp = configJson[full_name];
+        configJson[short_name] = tmp;
+        delete configJson[full_name];
+        repositoryNames[i] = short_name;
 
-        var checkChangesMd = configJson[name].checkChangesMd;
-        var remindForum = configJson[name].remindForum;
-        configJson[name].checkChangesMd = defined(checkChangesMd) ? parseBoolean(checkChangesMd) : true;
-        configJson[name].remindForum = defined(remindForum) ? parseBoolean(remindForum) : true;
+        if (!defined(configJson[short_name].gitHubToken)) {
+            return Promise.reject('repository ' + short_name + ' must have a `gitHubToken`');
+        }
+
+        var checkChangesMd = configJson[short_name].checkChangesMd;
```

Comment:
Our general rule is if you use a property more than once, store it in a local variable.  Instead of `configJson[short_name]` a bunch of times, you should only call it once.  Please review the rest of the code for this kind of pattern and clean it up as needed.  Thanks.